types/underscore/underscore-tests.ts

```diff
@@ -341,229 +359,233 @@ var buttonView = {
 
-_.mapObject({ a: 1, b: 2 }, val => val * 2) === _.mapObject({ a: 2, b: 4 }, _.identity);
-_.mapObject({ a: 1, b: 2 }, (val, key, o) => o[key] * 2) === _.mapObject({ a: 2, b: 4}, _.identity);
-_.mapObject({ x: "string 1", y: "string 2" }, 'length') === _.mapObject({ x: "string 1", y: "string 2"}, _.property('length'));
+// making a copy of an object that omits a specific subset of properties selected by unknown names
+_.omit({ name: 'moe', age: 50, userid: 'moe1' }, stringArray);
+
+// making a copy of an object that omits a specific subset of properties selected by an iteratee
+_.omit({ name: 'moe', age: 50, userid: 'moe1' }, (value, key) => key === 'name' || key === 'age');
+
+// converting the properties of an object from numbers to strings
+_.mapObject({ a: '1', b: '2' }, val => +val);
 
+// filling in properties missing on an object
 var iceCream = { flavor: "chocolate" };
 _.defaults(iceCream, { flavor: "vanilla", sprinkles: "lots" });
 
+// creating a shallow-copied clone of an object
 _.clone({ name: 'moe' });
 _.clone(['i', 'am', 'an', 'object!']);
 
+// move to chain tests
 _([1, 2, 3, 4])
     .chain()
     .filter((num) => { return num % 2 == 0; })
     .tap(alert)
     .map((num) => { return num * num; })
     .value();
 
-_.chain([1, 2, 3, 200])
-    .filter((num) => { return num % 2 == 0; })
-    .tap(alert)
-    .map((num) => { return num * num; })
-    .value();
-
+// checking whether or not an object has a property
 _.has({ a: 1, b: 2, c: 3 }, "b");
 
+// retrieving shallow and deep property values from an object
 var moe = { name: 'moe', luckyNumbers: [13, 27, 34] };
-
 _.property('name')(moe);
-_.property(['name'])(moe);
 _.property(['luckyNumbers', 2])(moe)
 
+// creating a function that will always return a specific value
 var UncleMoe = { name: 'moe' };
 _.constant(UncleMoe)();
 
+// getting the current time as an integer timestamp
 typeof _.now() === "number";
 
+// giving control of the _ global variable back to its previous owner (returns a reference to value of _ that is in effect before the function is called)
 var underscore = _.noConflict();
 
+// calling a no-op function that returns the same value that is used as the argument
 var moe2 = { name: 'moe' };
 moe2 === _.identity(moe);
 
-var genie;
-var r2 = _.times(3, (n) => { return n * n });
-_(3).times(function (n) { genie.grantWishNumber(n); });
+// calling a function multiple times with the iteration as an argument and getting an array containing the result of each call
+// in this case, the result is the squares of the numbers 0 through 4
+_.times(5, n => n * n);
 
+// generating a random number between two boundsW
 _.random(0, 100);
 
+// adding functions to Underscore by calling _.mixin and augmenting Underscore's
+// type definitions
 _.mixin({
     capitalize(string) {
         return string.charAt(0).toUpperCase() + string.substring(1).toLowerCase();
     }
 });
 (<any>_("fabio")).capitalize();
```

Comment:
This doesn’t really look like augmenting Underscore’s types, rather like working around the fact that you haven’t.

Reply:
I was planning to change this to https://github.com/DefinitelyTyped/DefinitelyTyped/compare/master...reubenrybnik:underscore-tests-usage-tests-cleanup-old#diff-4d35bcc89b6520b7b1930f73c7e9c4bfR558, but I ended up deciding to include that in subset A rather than doing it here for some reason. My best guess at the reason was to avoid excess changes that would make the diff look complicated, but I'm happy to move this change up to this subset.

Reply:
I've made the changes, and they don't appear to have made the diff unhappy, so it looks like I was overly cautious. Happy to have this in place 😄